include/gz/gui/qml/GzSpinBox.qml

```diff
@@ -18,12 +18,31 @@ import QtQuick 2.9
 import QtQuick.Controls 1.4
 import QtQuick.Controls.Styles 1.4
```

Comment:
For this, we should use the newer version of QtQuick Controls and remove the next line.

```suggestion
import QtQuick.Controls 2.15
import QtQuick.Controls.Styles 2.15
```

Reply:
Done.